src/search/search.component.ts

```diff
@@ -130,6 +130,7 @@ export class Search implements ControlValueAccessor {
 	 * Emits an event when value is changed.
 	 */
 	@Output() valueChange = new EventEmitter<string>();
+	@Output() opened = new EventEmitter<boolean>();
```

Comment:
Any reason why it can't be
```suggestion
	@Output() open = new EventEmitter<boolean>();
```
?

It seems like that would go more in line with our usual output naming.

Reply:
Ah that's a good point ... I'm not super picky, but `open` may make more sense yeah